roles/splunk_common/tasks/enable_admin_auth.yml

```diff
@@ -28,6 +28,11 @@
     become_user: "{{ splunk.user }}"
     no_log: "{{ hide_password }}"
 
+  - name: "Delete etc/passwd if exists (Windows)"
```

Comment:
This will actually apply to any OS. There's not condition set to limit it to Windows.
Might want to change the name to not mislead people.

Reply:
updating the ansible code to match up for windows only